dockhand-lru-registry/files/conf/registry-config.yaml

```diff
@@ -0,0 +1,20 @@
+version: 0.1
```

Comment:
it would be nice if the user had an option to merge their config onto this - in the case that they want to act as an upstream proxy or something.

we could make this whole thing a `toYaml` and define the default options in the chart's default `values.yaml`

Reply:
Added to values.yaml